examples_tests/__init__.py

```diff
@@ -162,7 +162,7 @@ def install_snap(self, example_dir, snap_name, version):
     def assert_command_in_snappy_testbed(self, command, expected_output):
         if not config.get('skip-install', False):
             output = self.run_command_in_snappy_testbed(command)
-            self.assertEqual(output, expected_output)
+            self.assertEqual(expected_output, output)
```

Comment:
with testtools, the expected is first. When it fails, it prints like: expected: first argument, actual: second argument. I always get it wrong, but I try to update it when I see it fail with the wrong message.

with only unittest I think it doesn't matter, because it prints something like: first is not equal to second.